bl-user-setup

```diff
@@ -18,15 +18,15 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # USER="$1" USER is exported by lightdm
```

Comment:
Turns out it's complicated.

The mapping behaviour of (user, domain) tuples to local user names  can be configured in Samba: https://www.samba.org/samba/docs/man/Samba-HOWTO-Collection/idmapper.html, and SSSD: https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Deployment_Guide/sssd-user-ids.html, very flexibly, and the user name in passwd could include e.g. "(a)" instead of "@":
```
getent passwd test.linksync(a)linkoffice.fr
```
So I think the correct solution would be to use whatever that lightdm passes over to us in "$USER" as we can't possibly know what the current PAM configuration gets its user names from. I've already prepared an improvement  branch here:  https://github.com/BunsenLabs/bunsen-configs/compare/improve-bl-user-setup and need to fix this exact part. I'll open a PR for you to review then.